interface/billing/print_daysheet_report_num1.php

```diff
@@ -68,1764 +68,1968 @@
+                                </td>
+                                <td class='text' width='125'>
+                                </td>
+                                <td class='text font-weight-bold' width='250'>
+                                    <?php echo xlt("Total Payments") . ': ' . "(%1\$.2f)", text($user_info['inspay'][$i] + $user_info['patpay'][$i]) ?>
+                                </td>
+                                <td class='text' width='25'>
+                                    <br />
+                                </td>
+
+                                <table class='table table-borderless'>
+                                    <tr>
+                                        <td class='text' width='25'>
+                                        </td>
+                                        <td class='text' width='250'>
+                                        </td>
+                                        <td class='text' width='125'>
+                                        </td>
+                                        <td class='text font-weight-bold' width='250'>
+                                            <?php echo xlt("Total Adj") . '.: ' . "(%1\$.2f)", text($user_info['patadj'][$i] + $user_info['insadj'][$i]); ?>
+                                        </td>
+                                        <td width='125'><span class='text'></span></td>
+                                        <td class='text font-weight-bold' width='250'>
+                                            <?php echo xlt("Refund") . ': ' . "(%1\$.2f)", text($user_info['patref'][$i] + $user_info['insref'][$i]) ?>
+                                        </td>
+                                        <td class='text' width='25'>
+                                            <br />
+                                        </td>
+                                        <table class='table table-borderless'>
+                                            <tr>
+                                                <td class='text' width='25'>
+                                                </td>
+                                                <td class='text' width='250'>
+                                                </td>
+                                                <td class='text' width='125'>
+                                                </td>
+                                                <td class='text' width='250'>
+                                                </td>
+                                                <td class='text' width='125'>
+                                                </td>
+                                                <td class='text font-weight-bold' width='175' height='5'>
+                                                    <hr noshade size='3'>
+                                                </td>
```

Comment:
Hey @tywrenn 

Should I remove the `noshade` attribute?

Reply:
Dunno what that even does so u can remove that lol